### PR TITLE
⬆️ Update Bookstack to v21.11.1

### PR DESCRIPTION
**Pull request by dm82m**
# Proposed Changes

- upgrade to v21.11.1

## Related Issues

### CODE REVIEW

**Comment by sinclairpaul** [comment]
Did you confirm that this actually builds and runs ok?

File: bookstack/DOCS.md [hunk 1]
```diff
@@ -127,9 +127,30 @@ MariaDB addon which should be installed prior to startup, this can be changed
 within the configuration to use an external MySql/MariaDB Database. Please note
 that there is no easy upgrade path between the two options.
 
+## SMTP Support
+
```
Comment (frenck) [inline bookstack/DOCS.md:131]: Would be nice to split this into a separate PR and add some context to the chapter.

Comment (dm82m) [reply bookstack/DOCS.md:131]: okay will remove it and file a new PR if I have the time for it

**Comment by dm82m** [comment]
> Did you confirm that this actually builds and runs ok?

no, I would test it out if you explain how to test that. I just requested the upgrade but issue got closed with the ask for creating a PR. thats what I did. I had the hope that CI will check if the change was successful. but anyways: if you point me to the way how I could test that locally I would go for a try ...

**Comment by sinclairpaul** [comment]
There are dependency updates, therefore this won't actually build, we also have other base image updates that need to be completed.

I will get to this at some point this week, thanks for the effort, however for now I will close this as there is more work that needs to be performed.

**Comment by dm82m** [comment]
thanks for your help @sinclairpaul 

sorry for the will to learn and help @frenck, will keep my fingers away in the future, have a nice day

**Comment by frenck** [comment]
> sorry for the will to learn and help @frenck, will keep my fingers away in the future, have a nice day

👍 You don't have to be sorry for that... I'm just saying putting out random untested PRs is generally not something one should do. The least you could have done is state you have no clue on how to test it and would like some help with that. However, that isn't in the PR description.

Don't get me wrong, help is appreciated, if responsible.
